frontend/rpm/template.go

```diff
@@ -226,7 +232,10 @@ func (w *specWrapper) PrepareSources() (fmt.Stringer, error) {
 				return nil
 			}
```

Comment:
would we still need to handle when the case is a patch and a directory when preparing sources?

Reply:
We just don't support that right now and it will give an error.
We can handle this in a follow-up.